### ISSUE TITLE
Weird corners in KDE Plasma 5.25 Beta

### ISSUE THREAD

**Issue by kun-codes** [Shaurya-Kalia/breezeblurredplasmatheme#1]
The default breeze theme works perfectly with the new floating panel option ticked. Let me know if you want images which show the differences between them.

**Comment by Shaurya-Kalia** [comment]
Yes, please share the images. I will build kde 5.25 too in the mean time to test it. I haven't followed the changes to breeze plasma theme in kde 5.25, but it if has had changes to it then I would need to update this one

**Comment by kun-codes** [comment]
I will share the images by tomorrow

**Comment by kun-codes** [comment]
this is the default breeze theme https://imgur.com/a/nkGsUjY
and this is the breeze blurred plasma theme https://imgur.com/a/4ddXVp4

there are two white corners at each side of the floating panel

**Comment by Shaurya-Kalia** [comment]
i guess it is due to the change in transparency mask radius in 5.25. Currently, breeze blurred is based off breeze 5.24 . I will try to update it before 5.25 is released

**Comment by kun-codes** [comment]
Alright, let me know when it is fixed

**Comment by kun-codes** [comment]
@Shaurya-Kalia plasma 5.25 has been released on arch linux, any updates on [breezeblurredplasmatheme](https://github.com/Shaurya-Kalia/breezeblurredplasmatheme)?

**Comment by Shaurya-Kalia** [comment]
I added modified theme. Please check it out. Due to lower level changes in plasma 5.25, it may not exactly match in appearance. However I tried to keep it as close as possible

**Comment by kun-codes** [comment]
I updated via discover

```
❯ lsd $HOME/.local/share/plasma/desktoptheme
 breeze_blurred_2

```
I have the correct theme with me, but the same problem exists on kde plasma 5.25

**Comment by Shaurya-Kalia** [comment]
I had some strange behaviour with discover. Remove the existing theme(s) and then reinstall it.

**Comment by Shaurya-Kalia** [comment]
Could you attach screenshot if its still showing the problem?

**Comment by kun-codes** [comment]
I removed the discover theme and removed it from settings panel manually, but now it shows this if I install it via discover
![image](https://user-images.githubusercontent.com/77796630/174424377-8585630b-276d-4702-b766-0199d73b07e8.png)

**Comment by Shaurya-Kalia** [comment]
Well, its a bug on pling or kde's get new stuff component's end, which I can't do much about. Until then, you can manually download the tar.gz file from kde store or here and install it through "install from file" button on plasma theme settings page
